lib/cjs.ts

```diff
@@ -13,10 +13,9 @@ import cjsPlatformShim from './platform-shims/cjs.js';
 
 // See https://github.com/yargs/yargs#supported-nodejs-versions for our
 // version support policy. The YARGS_MIN_NODE_VERSION is used for testing only.
-const minNodeVersion =
-  process && process.env && process.env.YARGS_MIN_NODE_VERSION
-    ? Number(process.env.YARGS_MIN_NODE_VERSION)
-    : 10;
+const minNodeVersion = process?.env?.YARGS_MIN_NODE_VERSION
+  ? Number(process.env.YARGS_MIN_NODE_VERSION)
+  : 10;
```

Comment:
perhaps in a second PR, we should actually make this `: 12` now, since we just dropped Node 10 from the engines field